BricksAndHearts.UnitTests/ServiceTests/Admin/AdminServiceTests.cs

```diff
@@ -72,4 +75,169 @@ public void CancelAdminAccessRequest_SetsHasRequestedAdminToFalse_ForCorrectUser
         // Assert
         context.Users.Single(u => u.Id == requestedAdminUser.Id).HasRequestedAdmin.Should().BeFalse();
     }
+
+    [Fact]
+    public void GetAdminLists_GetsListOfCurrentAndPendingAdmins_ForAdminUser()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetAdminLists().Result;
+
+        // Assert
+
+        result.Item1.Should().BeOfType<List<UserDbModel>>();
+        result.Item2.Should().BeOfType<List<UserDbModel>>();
+    }
+
+    [Fact]
+    public void GetLandlordDisplayList_CalledWithApproved_ReturnsApprovedLandlords()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetLandlordDisplayList("Approved").Result;
+
+        // Assert
+        result.Should().BeOfType<List<LandlordDbModel>>().And.Subject.Where(u => u.CharterApproved).Should()
+            .HaveCount(result.Count);
+    }
+    
+    [Fact]
+    public void GetLandlordDisplayList_CalledWithUnapproved_ReturnsUnapprovedLandlords()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetLandlordDisplayList("Unapproved").Result;
+
+        // Assert
+        result.Should().BeOfType<List<LandlordDbModel>>().And.Subject.Where(u => u.CharterApproved == false).Should()
+            .HaveCount(result.Count);
+    }
+    
+    [Fact]
+    public void GetLandlordDisplayList_CalledWithNothing_ReturnsAllLandlords()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetLandlordDisplayList("").Result;
+        
+        // Assert
+        result.Should().BeOfType<List<LandlordDbModel>>().And.Subject.Should().HaveCount(context.Landlords.ToList().Count);
+    }
+
+    [Fact]
+    public void FindUserById_GivenCorrectId_FindsUser()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+        var approvedLandlord = Fixture.CreateApprovedLandlordUser();
+        
+        // Act
+        var result = service.FindUserByLandlordId(1);
+
+        // Assert
+
+        result.Should().BeOfType<UserDbModel>();
+        result!.GoogleAccountId.Should().Be(approvedLandlord.GoogleAccountId);

```

Comment:
I found this a bit confusing! Makes sense now but worth a comment

```suggestion
        // Arrange
        using var context = Fixture.CreateReadContext();
        var service = new AdminService(context);
        
        // Act
        var result = service.FindUserByLandlordId(1);

        // Assert
        result.Should().NotBeNull();
        // This landlord is created first by the fixture so should have id 1
        var approvedLandlord = Fixture.CreateApprovedLandlordUser()
        result!.GoogleAccountId.Should().Be(approvedLandlord.GoogleAccountId);
```